test_redirects/map_globalconf.py

```diff
@@ -213,8 +213,8 @@
     url_test('/firefox/mobile/faq/?os=firefox-os', '/firefox/os/faq/'),
 
     # Bug 986174
-    url_test('/firefox/mobile/features/', '/firefox/android/'),
-    url_test('/firefox/mobile/faq/', '/firefox/android/faq/'),
+    url_test('/{m,{firefox/,}mobile}/features/', '/firefox/android/'),
+    url_test('/{m,{firefox/,}mobile}/faq/', '/firefox/android/faq/'),
```

Comment:
Sorry, just saw this. You're right in your 2nd comment. I know it's a bit confusing when nested like that, and I could write it out fully, but this does work and makes sense to me, and I think will to others if they know bash expansion and regex. Also [braceexpand](https://pypi.python.org/pypi/braceexpand) is a wonderful tool.